src/current/v24.3/triggers.md

```diff
@@ -508,6 +508,12 @@ In the following example, a combination of `BEFORE` and `AFTER` triggers is used
 	1. The row value fails the conditional check in `ensure_minimum_wage`, and raises an exception.
 	1. The `ERROR` message is printed and the SQL operation is aborted before the `give_bonus` function is executed.
 
+### Video demo
+
+For a deep-dive demo on triggers, play the following video:
```

Comment:
nit, for consistency with references to other video links:
```suggestion
For a deep-dive demo on triggers, watch the following video:
```

Reply:
I changed the wording here for the same reason others have suggested "refer to" over "see" (accessibility).